src/test/java/com/EighthLight/app/TestConfig.java

```diff
@@ -17,15 +17,17 @@
     private ArrayList<String> gameModeTwoUserInputs;
     private ArrayList<String> gameModeIncorrectInput;
     private ArrayList<String> boardSizeIncorrectInput;
+    private ArrayList<String> playerOrderingTwoUserInputs;
 
     @BeforeEach
     public void setUp() {
-        defaultUserInputs = new ArrayList<>(Arrays.asList("X", "O", "1", "3", ""));
-        duplicateSymbolUserInputs = new ArrayList<>(Arrays.asList("X", "X", "O", "1", "3"));
-        gameModeOneUserInputs = new ArrayList<>(Arrays.asList("X", "O", "1", "3"));
-        gameModeTwoUserInputs = new ArrayList<>(Arrays.asList("X", "O", "2", "3"));
-        gameModeIncorrectInput = new ArrayList<>(Arrays.asList("X", "O", "5", "1", "3"));
-        boardSizeIncorrectInput = new ArrayList<>(Arrays.asList("X", "O", "1", "Ten", "3"));
+        defaultUserInputs = new ArrayList<>(Arrays.asList("X", "O", "1", "1", "1", "3"));
```

Comment:
At this point, I think it's probably more readable to make these local variables in the tests? I could be convinced otherwise.